charts/spire/templates/_spire-lib.tpl

```diff
@@ -80,17 +80,50 @@
 {{-   $type }}
 {{- end }}
 
+{{/* Takes in a dictionary with keys:
+ * ingress - the standardized ingress object
+ * Values - Chart values
+*/}}
+{{ define "spire-lib.ingress-calculated-name" }}
+{{- $host := .ingress.host }}
+{{- if not (contains $host ".") }}
+{{-   $host = printf "%s.%s" $host (include "spire-lib.trust-domain" .) }}
+{{- end }}
+{{- $host }}
+{{- end }}
+
 {{/* Takes in a dictionary with keys:
  * ingress - the standardized ingress object
  * svcName - The service to route to
  * port - which port on the service to use
+ * Values - Chart values
```

Comment:
What is Values being used for? I don't see it referenced below.

Reply:
Its a little indirect....
Added at line 102:
{{- $host := include "spire-lib.ingress-calculated-name" . }}
Which calls at line 90:
(include "spire-lib.trust-domain" .)

Which wants .Values for so it can determine what the trustDomain is, either subchart trustDomain setting or global one.

Never needed full Values here before.

Reply:
Thanks for clarifying.